docs/conch/examples/sshsimpleclient.py

```diff
@@ -22,6 +14,15 @@
 $ ckeygen -t rsa -f ssh-keys/client_rsa
 """
```

Comment:
This change is just … incorrect? It is changing an inline comment to a docstring.